Formula/makensis.rb

```diff
@@ -32,8 +39,13 @@ def install
     # https://sourceforge.net/p/nsis/bugs/1085/
     ENV.libstdcxx if ENV.compiler == :clang
 
+    # requires zlib (win32) to build utils
+    resource("zlib-win32").stage do
```

Comment:
@dunn Shall I use `FileUtils.chmod` or does Homebrew provide methods specifically for this (didn't find anything in the [documentation](http://www.rubydoc.info/github/Homebrew/brew/master))